public_html/wp-content/mu-plugins/blocks/view/shared/featured-image.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace WordCamp\Blocks\Shared\Components;
+
+/**
+ * Provides render backend for FeaturedImage component.
+ *
+ * @param array    $class_names        Additional classes to add inside <img> tag.
+ * @param \WP_Post $post               Current post object. This will be used to calculate srcset attribute.
+ * @param string   $selected_image_url URL for selected sized image.
+ * @param int      $height             Height of the image
+ * @param int      $width              Width of the image
+ *
+ * @return string Output markup for featured image.
+ */
+function render_featured_image( $class_names, $post, $height, $width ) {
```

Comment:
@vedanshujain , this doesn't feel like a "view" to me, since it doesn't have any actual HTML. I wonder if it'd be more logical to live in `blocks/includes/shared.php`?

Reply:
Hmm, yeah thats a good idea. Will move it there.

Reply:
Being fixed in https://github.com/WordPress/wordcamp.org/pull/66 .